### DISCUSSION TITLE
What data can be publicly viewable?

### DISCUSSION THREAD

**Discussion by Christopher-Hayes** [tweetback/tweetback#28] [Q&A] [answered]
Is all the Twitter data placed in `tweets.js` able to be publicly viewable without any security/privacy issues? Just wondering if I can fork and push to a public repo?

**Comment by Accudio** [comment, marked as answer]
For the most part yes, `tweets.js` only contains your Tweets and Retweets, nothing sensitive that isn't otherwise public.
The exceptions are if you have a private twitter account or if you use Twitter Circles—tweets to your circles are also listed.

**Reply by Christopher-Hayes** [reply]
Gotcha, thanks!
